resources/knative-eventing/values.yaml

```diff
@@ -1,2 +1,8 @@
 global:
   isLocalEnv: false
+
+knative-eventing:
+  tracing:
+    enable: true
```

Comment:
is there a way to get this value from kymas istio configuration? It should be the same value across all kyma

Reply:
Definitely, there is a way by using "overrides". But that will bloat the PR scope IMO and we have to holistically solve it for all the components e.g. publish-service, console-backend-service. How about solving it in a different issue?